src/app/header/product/product.component.scss

```diff
@@ -0,0 +1,85 @@
+.col-2 {
```

Comment:
If it requires to override common classes, we should be using custom classes

Reply:
done